main.gms

```diff
@@ -230,548 +217,1220 @@ $setGlobal c_description  REMIND run with default settings
+*** (KWlikeItrCPnash): Internalize damage function based on Kalkuhl & Wenz (2020), but with Nash SCC, i.e. each region only internalizes its own damages. Requires cm_emiscen set to9 for now.
+*** (KWlikeItrCPreg): Internalize damage function based on Kalkuhl & Wenz (2020), but with regional SCC instead of a global uniform price. Requires cm_emiscen set to 9 for now.
+*** (KW_SEitr): Internalize damage function based on Kalkuhl & Wenz (2020) for upper limit based on standard error. Requires cm_emiscen set to 9 for now.
+*** (KWTCintItr): Internalize combined damages from Kalkuhl & Wenz (2020) and from tropical cyclones. Requires cm_emiscen set to 9 for now.
+*** (LabItr): Internalize labor supply damages based on Dasgupta et al. (2021). Requires cm_emiscen set to 9 for now.
+*** (TCitr): Internalize tropical cyclone damage function based on Krichene et al. (2022). Requires cm_emiscen set to 9 for now.
 $setGlobal internalizeDamages  off               !! def = off
 ***---------------------    70_water  -------------------------------------------
+*** (off): no water demand taken into account
+*** (exogenous): exogenous water demand is calculated based on data on water demand coefficients and cooling shares
+*** (heat): as exogenous only that vintage structure in combination with time dependent cooling shares as vintages and efficiency factors are also considered and demand is a function of excess heat as opposed to electricity output
 $setglobal water  off                 !! def = off
 ***---------------------    80_optimization    ----------------------------------
+*** (nash): Nash solution. Adjust cm_nash_autoconverge to needs.
+*** (negishi): calculates a Negishi solution (social planner)
+*** (testOneRegi):  solves the problem for one region for given prices (taken from gdx).
+***                ! Warning:  For development purposes only !
 $setGlobal optimization  nash         !! def = nash
 ***---------------------    81_codePerformance    -------------------------------
+*** (off): nothing happens
+*** (on):  test code performance: noumerous (30) succesive runs performed in a triangle, tax0, tax30, tax150, all growing exponentially,
+***                    therefore use carbonprice|exponential, c_emiscen|9, and cm_co2_tax_2020|0.
 $setGlobal codePerformance  off       !! def = off
 
 ***-----------------------------------------------------------------------------
-***                     SWITCHES and FLAGS
+***                     SWITCHES 
 ***-----------------------------------------------------------------------------
 ***--------------- declaration of parameters for switches ----------------------
-parameters
+parameter
   cm_iteration_max          "number of iterations, if optimization is set to negishi or testOneRegi; used in nash mode only with cm_nash_autoconvergence = 0"
+;
+  cm_iteration_max       = 1;     !! def = 1
```

Comment:
```suggestion
parameter
  cm_iteration_max          "number of iterations, if optimization is set to negishi or testOneRegi; used in nash mode only with cm_nash_autoconvergence = 0"
         / 1 /    !! def = 1
;
```
Remind me again what the reason was to do the definition and the starting value in two different statements. Is ist because `gms` and/or `lucode2` can't deal with the combined definition?

Reply:
tradition? I also wanted to change to your suggestion but wanted to keep the current version for now. We anyway need to come up with a template how to organize parameter description/documentation in main.gms.